html/captive-portal/lib/captiveportal/PacketFence/DynamicRouting/Application.pm

```diff
@@ -388,7 +397,7 @@ sub _render {
 
     our $processor = Template::AutoFilter->new($TT_OPTIONS);;
     my $output = '';
-    $processor->process($template, $args, \$output) || die("Can't generate template $template: ".$processor->error."Error : ".$@);
+    $processor->process($template, $args, \$output, {binmode => ':utf8'}) || die("Can't generate template $template: ".$processor->error."Error : ".$@);
```

Comment:
This is not handled yet.

$stash->{cmd1} = ["Kneel before %s", "Zod"];
$stash->{cmd2} = "Kneel before Zaymitte";

[% i18n(cmd1)%] vs [% i18n(cmd2)%]